### PR TITLE
fix(ga4): update hybrid mode check condition and remove dependency on…

### PR DESCRIPTION
**Pull request by mihir-4116**
… useNativeSDKToSend flag

## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/v0/util/index.js [hunk 1]
```diff
@@ -1681,12 +1681,7 @@ const validatePhoneWithCountryCode = (phone) => {
  * @param {*} Config
  * @returns
  */
-const isHybridModeEnabled = (Config) => {
-  if (isDefinedAndNotNull(Config.useNativeSDK) && isDefinedAndNotNull(Config.useNativeSDKToSend)) {
-    return Config.useNativeSDK && !Config.useNativeSDKToSend;
-  }
-  return false;
-};
+const isHybridModeEnabled = (Config) => Config.connectionMode === 'hybrid';
```
Comment (koladilip) [inline src/v0/util/index.js:1684]: DB configs moved this new value?

Comment (mihir-4116) [reply src/v0/util/index.js:1684]: no, `connectionMode` was already coming but we were not using it. now as we decided to check hybridmode condition based on `connectionMode` key we will use that one

Comment (koladilip) [reply src/v0/util/index.js:1684]: is it true for any destination?

Comment (mihir-4116) [reply src/v0/util/index.js:1684]: only the one who is in new form builder and this is only recommended to use for only new form builder destinations

**Review by koladilip — APPROVED** [review]

**Review by sandeepdsvs — APPROVED** [review]

**Pull request merged**